src/compiler/factory/emitHelpers.ts

```diff
@@ -907,11 +849,10 @@ namespace ts {
         importName: "__classPrivateFieldGet",
         scoped: false,
         text: `
-            var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (receiver, privateMap) {
-                if (!privateMap.has(receiver)) {
-                    throw new TypeError("attempted to get private field on non-instance");
-                }
-                return privateMap.get(receiver);
+            var __classPrivateFieldGet = (this && this.__classPrivateFieldGet) || function (receiver, state, kind, f) {

```

Comment:
These helpers will be very hard to read without a reference document like Ron's gist. I guess we can ask how they want to go about it.

Reply:
I can copy the comments in the gist to the source code. Other helpers have extensive comments